### PR TITLE
WIP Ember Ignite announcement for Ember Times

### PR DESCRIPTION
**Pull request by jenweber**
WIP - do not merge this until after the organizers have commented that they are ready for the announcement to go out.

### CODE REVIEW

**Review by ChaiWithJai — APPROVED** [review]
OMG, wow this is amazing! Thank you so, so much, @jenweber for being so kind and helpful.

File: content/the-ember-times-issue-194.md [hunk 1]
```diff
@@ -44,6 +44,19 @@ In [another episode](https://www.whiskeywebandwhatnot.fm/robbies-origin-story-le
...
+## [Ember Ignite virtual mini conference 🔥](TODO)
+
+Ember Ignite is an international virtual mini conference scheduled for Wednesday, February 23rd!
+
+Many teams talk about the challenges of hiring and training Ember developers, and this conference tackles those challenges directly.
+Attendees will level up their training and teaching skills in order to build strong teams.
+
+Tickets are donation-based with a suggested amount of $30 USD. Proceeds will go towards creating an open source curriculum for teaching Ember to junior developers and taking a live cohort of 10 junior engineers (with no experience) and teaching them Ember so well, that they'll go toe-to-toe against mid-level UI engineer candidates. This effort is led by [Jai Bhagat](https://twitter.com/ChaiWithJai).
+
+[Register here](TODO) today!
```
Comment (ChaiWithJai) [inline content/the-ember-times-issue-194.md]: The event link is:  https://www.eventbrite.com/e/ember-ignite-tickets-232969998537

I think with your suggestions (fully incorporated) re:  landing page copy, the description is much more clear.

**Comment by jenweber** [comment]
@ChaiWithJai should this go out today or in 2 more weeks? Please tag @jaredgalanis with your answer to let him know.

File: content/the-ember-times-issue-194.md (hunk 1 above)
Comment (jenweber) [reply content/the-ember-times-issue-194.md]: Added the link!

**Review by ChaiWithJai — COMMENTED** [review]
This is great to go out today! We're < 50 days out so its a perfect time to promote.

**Review by jaredgalanis — APPROVED** [review]
Thank you!

**Pull request merged**